godtools/Managers/LanguagesManager.swift

```diff
@@ -43,7 +43,7 @@ class LanguagesManager: NSObject {
     func loadFromRemote() -> Promise<[Language]> {
         showNetworkingIndicator()
         
-        return Alamofire.request(URL(string: "\(GodToolsConstants.kApiBase)/\(path)")!).responseData().then { data -> Promise<[Language]> in
+        return Alamofire.request(URL(string: "\(GTConstants().kApiBase)/\(path)")!).responseData().then { data -> Promise<[Language]> in
```

Comment:
I thought I should be able to do that, but it gave me a build error the first time. It works now though. Thanks!